sdk/resourcemanager/Azure.ResourceManager.Core/src/ArmClient.cs

```diff
@@ -155,6 +155,15 @@ public virtual ResourceGroupOperations GetResourceGroupOperations(ResourceGroupR
             return new ResourceGroupOperations(new SubscriptionOperations(new ClientContext(ClientOptions, Credential, BaseUri, Pipeline), id.SubscriptionId), id.ResourceGroupName);
         }
 
+        /// <summary>
+        /// Gets the predefined tag operations under this subscription.
+        /// </summary>
+        /// <returns> The predefined tags operations. </returns>
+        public virtual PreDefinedTagOperations GetPreDefinedTagsOperations()
```

Comment:
I think this should be on the subscriptionOperations since I might want to adjust the pre defined tags for many different subscriptions

Reply:
Done